south-african-journal-of-geology.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>South African Journal of Geology</title>
+    <title-short>Geol. South Africa</title-short>
+    <id>http://www.zotero.org/styles/south-african-journal-of-geology</id>
+    <link rel="self" href="http://www.zotero.org/styles/south-african-journal-of-geology"/>
```

Comment:
@rmzelle & @adam3smith, I had started with meteorological-applications.csl as a base and intended to only make minor edits but soon found that the "minor" where more pervasive that originally anticipated. I then borrowed from the-geological-society-of-america.csl, the-geological-society-of-london.csl, APA,csl and harvard-anglia-ruskin-university.csl. By the time I was finished I wasn't sure where some the code came from and how much of the base was left.